<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotUserInterface</class>
 <widget class="QWidget" name="RobotUserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RobotUserInterface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
color: rgb(255, 255, 255);
background-color: #00000000;
}
QScrollArea{
border: 0px solid;
border-right-width: 0px;
background-color: #00000000;
}
QScrollBar:vertical {
border: none;
background: rgba(255, 255, 255,0);
width: 6px;
margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
background: rgb(70, 70, 70);
min-height: 20px;
border-radius: 2px;
border: none;
}
QScrollBar::handle:vertical::hover {
background: rgb(120, 120, 120);
min-height: 20px;
border-radius: 2px;
border: none;
}
QScrollBar::handle:vertical::pressed {
background: rgb(100, 100, 100);
min-height: 20px;
border-radius: 2px;
border: none;
}
QScrollBar::add-line:vertical {
border: 0px solid grey;
background: #32CC99;
height: 0px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
border: 0px solid grey;
background: #32CC99;
height: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;
width: 0px;
height: 0px;
}
QScrollBar:horizontal {
border: none;
background: rgba(255, 255, 255,0);
height: 6px;
margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
background: rgb(70, 70, 70);
min-height: 20px;
border-radius: 2px;
border: none;
}
QScrollBar::handle:horizontal::hover {
background: rgb(120, 120, 120);
min-height: 20px;
border-radius: 2px;
border: none;
}
QScrollBar::handle:horizontal::pressed {
background: rgb(100, 100, 100);
min-height: 20px;
border-radius: 2px;
border: none;
}
QScrollBar::add-line:horizontal {
border: 0px solid grey;
background: #32CC99;
height: 0px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
border: 0px solid grey;
background: #32CC99;
width: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
background: none;
width: 0px;
height: 0px;
}</string>
  </property>
  <layout class="QGridLayout" name="LayoutMain">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page_operation">
      <layout class="QGridLayout" name="layout_operation">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_info">
      <layout class="QGridLayout" name="layout_info">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_curve">
      <layout class="QGridLayout" name="layout_curve">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_settings">
      <layout class="QVBoxLayout" name="layout_settings">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1000</width>
            <height>380</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="layout_obj_settings"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QtMaterialRaisedButton" name="widget_settings_reset" native="true">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QtMaterialRaisedButton" name="widget_settings_apply" native="true">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_tools">
      <layout class="QGridLayout" name="layout_fileCatch">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QWidget</extends>
   <header>qt_material_widgets/qtmaterialraisedbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../RUI.qrc"/>
 </resources>
 <connections/>
</ui>
